manifests/02-deployment.yaml

```diff
@@ -76,3 +81,10 @@ spec:
       - name: metrics-tls
         secret:
           secretName: metrics-tls
+      - name: trusted-ca
+        configMap:
+          name: trusted-ca
+          optional: false
```

Comment:
Do we need this?  I'd expect `false` to be the default, although it's not spelled out [in the docs][1].

[1]: https://search.svc.ci.openshift.org/?name=proxy-4.2&search=waiting%20for%20Kubernetes%20API:%20context%20deadline%20exceeded

Reply:
@wking I believe `false` is the default. I thought it would be helpful to be explicit. I'll remove.